Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -6745,7 +6741,7 @@ class ConstructTarget_2 {
   \ingroup PkgKernel23ConceptsFunctionObjects
   \cgalConcept
 
-  \cgalRefines `AdaptableFunctor` (with one argument)
+  \cgalRefines{AdaptableFunctor (with one argument)}
```

Comment:
I now understand it, but I think this will be obsolete / not relevant anymore as soon as we replace the functor "with ... argument"  with the relevant `Adaptable...Function` (good point for the search action though).